shared/util/kbfs-notifications.js

```diff
@@ -162,6 +162,7 @@ export function kbfsNotification(notification: FSNotification, notify: any, stat
     // independently.
     [kbfsCommonFSNotificationType.initialized]: '',
     [kbfsCommonFSNotificationType.connection]: '',
+    [kbfsCommonFSNotificationType.syncConfigChanged]: 'Synchronization config changed',
```

Comment:
Isn't this going to give notifications even if you just changed the config directly from the UI? Also this text isn't really clear. I'd say for now we can just make it blank.

Reply:
I'll make it blank.

Reply:
That is keep it commented out so it will be not displayed.